delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/OrganizationsRoutes.scala

```diff
@@ -100,7 +98,7 @@ final class OrganizationsRoutes(identities: Identities, organizations: Organizat
               operationName(s"$prefixSegment/orgs/{label}") {
                 concat(
                   put {
-                    parameter("rev".as[Long]) { rev =>
+                    parameter("rev".as[Int]) { rev =>
```

Comment:
Is this something that we have to announce as part of breaking changes?

Reply:
I think we should focus on the real ones like SSEs for example.

It does not change anything in the API per se and nobody has or will reach the 2.1B limit